### ISSUE TITLE
change select state label on canadian school finder to select province

### ISSUE THREAD

**Issue by mikefantini** [DoSomethingArchive/legacy-website#3320]
![image](https://cloud.githubusercontent.com/assets/6330971/4912229/69ac5faa-649a-11e4-8741-a876c5bee934.png)

**Comment by aaronschachter** [comment]
Deploying International QA v0.3.84.  Current version was v0.3.78 

`dosomething_canada_update_7006` is not being executed on deploy because of Message Broker table issues:

```
DEBUG [98653917]    Updating database...
DEBUG [98653917]     Message_broker_p  7002  Implements hook_update_xx().   Delete all system       
DEBUG [98653917]     roducer                 variables related to the previous version. Create      
DEBUG [98653917]                             message_broker_producer_productions database table.    
DEBUG [98653917]     Dosomething_cana  7006  Updates                                                
DEBUG [98653917]     da                      dosomething_user_address_administrative_area_options() 
DEBUG [98653917]                             for CA site.   New variable is an array, including     
DEBUG [98653917]                             title ("Province").                                    
DEBUG [98653917]     Dosomething_fact  7005  Removes field_intro, field_intro_title,                
DEBUG [98653917]     _page                   field_hero_image from fact_page.                       
DEBUG [98653917]     Dosomething_fact  7006  Sets dosomething_fact_page_copy_intro variable.        
DEBUG [98653917]     _page                                                                          
DEBUG [98653917]     Dosomething_mbp   7002  Creates default production entries in                  
DEBUG [98653917]                             message_broker_producer                                
DEBUG [98653917]     Dosomething_user  7005  Updates                                                
DEBUG [98653917]                             dosomething_user_address_administrative_area_options() 
DEBUG [98653917]                             for US site.   New variable is an array, including     
DEBUG [98653917]                             title ("State").
DEBUG [98653917]    Do you wish to run all pending updates? (y/n): y
DEBUG [98653917]    Performed update: dosomething_fact_page_update_7005                  [ok]
DEBUG [98653917]    Performed update: dosomething_user_update_7005                       [ok]
DEBUG [98653917]    PHP Fatal error:  Call to undefined function message_broker_producer_production_create() in /var/www/international.dosomething.org/releases/20141105194006/lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install on line 80
DEBUG [98653917]    
DEBUG [98653917]    Fatal error: Call to undefined function message_broker_producer_production_create() in /var/www/international.dosomething.org/releases/20141105194006/lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install on line 80
DEBUG [98653917]    Drush command terminated abnormally due to an unrecoverable error.   [error]
DEBUG [98653917]    Error: Call to undefined function
DEBUG [98653917]    message_broker_producer_production_create() in
DEBUG [98653917]    /var/www/international.dosomething.org/releases/20141105194006/lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install,
DEBUG [98653917]    line 80
DEBUG [98653917]    The external command could not be executed due to an application     [error]
DEBUG [98653917]    error.
DEBUG [98653917]    WD varnish: Authentication to server failed!                         [error]
DEBUG [98653917]    WD varnish: Recieved status code 101 running ban req.http.host ~     [error]
DEBUG [98653917]    canada && req.url ~ "/". Full response text: Unknown request.
DEBUG [98653917]    Type 'help' for more info.
DEBUG [98653917]    
DEBUG [98653917]    
DEBUG [98653917]    WD varnish: Authentication to server failed!                         [error]
DEBUG [98653917]    WD varnish: Recieved status code 101 running ban req.http.host ~     [error]
DEBUG [98653917]    canada && req.url ~ "/". Full response text: Unknown request.
DEBUG [98653917]    Type 'help' for more info.
```

Current Canada schema is 7005, which is causing this bug.

**Comment by aaronschachter** [comment]
This may be because the modules are disabled but not uninstalled (module updates still run if a module is disabled).  What's a bit more concerning here is if our US deploy will bork upon next deploy.

**Comment by aaronschachter** [comment]
Testing. Uninstalled MBP and Message Broker on QA, manually updated at update.php, and it ran the updates. The dropdown label and options have been updated on QA... uninstalling the modules on Canada prod may fix this.

**Comment by aaronschachter** [comment]
This is because https://github.com/DoSomething/dosomething/commit/ccb7e184b45dec7869fca3b50abbd1d9d5a26079 has not been deployed to Canada production yet.  

I've deployed to QA Canada to test and confirmed there are correct label/provinces:https://qa.canada.dosomething.org/node/1227 although this will be overwritten tomorrow when the QA Canada DB is refreshed. 

We'll deploy international tomorrow AM, upon which Canada QA will be updated in subsequent databse pulls. 

Closing this, as its been resolved by https://github.com/DoSomething/dosomething/pull/3298
